instrumentation/Makefile

```
@@ -70,7 +70,11 @@ docker-run:
 	docker run --rm -it -v `pwd`:/instr instr-devel
 
 .PHONY: tests
-tests: test-java test-nodejs
+tests: test-java test-nodejs test-dotnet
+
+.PHONY: test-dotnet
+test-dotnet: dist
+	(cd tests/dotnet && ARCH=$(ARCH) ./test.sh)
```

A pattern rule can replace the other two targets:

```suggestion
.PHONY: test-dotnet-java-nodejs
test-%: dist
	(cd tests/$* && ARCH=$(ARCH) ./test.sh)
```

I'm not sure of the phony convention mechanics but that's what is used elsewhere in our makefiles.

---

sure, thanks!

---

You can delete the following two targets (now dupes)